src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,15 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
+    private static final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User potentialUser = userService.findByEmail(email);
+        if (potentialUser == null) {
+            return false;
+        } else {
+            return potentialUser.getPassword().equals(password);
+        }
```

Comment:
```suggestion
        } 
        return potentialUser.getPassword().equals(password);
```
we don't need else after return